Repositories/ExcelExport.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Leantime\Plugins\ExcelExport\Repositories;
+
+use Illuminate\Contracts\Container\BindingResolutionException;
+use Leantime\Domain\Timesheets\Repositories\Timesheets as TimesheetRepository;
+
+
+/**
+ * ExcelExport Repository
+ */
+class ExcelExport {
+  private TimesheetRepository $timesheetRepository;
+
+  /**
+   * constructor
+   *
+   * @access public
+   *
+   */
+  public function __construct(
+    TimesheetRepository $timesheetRepository,
+  ) {
+    $this->timesheetRepository = $timesheetRepository;
+  }
```

Comment:
Use https://stitcher.io/blog/constructor-promotion-in-php-8 here:

```suggestion
  /**
   * constructor
   *
   * @access public
   *
   */
  public function __construct(
    private readonly TimesheetRepository $timesheetRepository,
  ) {
  }
```